flock_agent/daemon/daemon.py

```diff
@@ -155,16 +181,24 @@ def log(self, request, response, time):
             except:
                 return response_object(error="invalid twig_id")
 
-        async def enable_twig(request):
-            twig_id = await request.json()
-            self.global_settings.enable_twig(twig_id)
-            self.global_settings.save()
-            return response_object()
+        async def enable_undecided_twigs(request):
```

Comment:
Can you comment what this function is for? I understand what it does, but not exactly why we'd be in a situation where it's necessary.

Reply:
This is for if users choose to automatically opt in to new twigs without being asked if they want to opt-in (which is checked by default in the onboarding wizard). If this is the case, and they install a Flock Agent update that includes new twigs, then those twigs start out as undecided, and this function opts them in. Basically there was logic in the GUI that did this, but I moved the logic to daemon here.